spec/factories/case/offender_sars.rb

```diff
@@ -30,6 +30,8 @@
     other_subject_ids               { 'ABC 123 DEF' }
     case_reference_number           { '123 ABC 456' }
     subject_address                 { '22 Sample Address, Test Lane, Testingington, TE57ST' }
+    request_dated { Date.parse('13-07-2010') }
+    requester_reference { '456 ABC 123' }
```

Comment:
I wouldn't hold up anything on account of it, but FWIW...

I don't normally like this approach (like on lines 30-32) aligning things with spaces in vertical rows as if they're tabbed columns  - wouldn't do it myself - but if they're lined up when we encounter the file, I'd maintain the arrangement

Wouldn't recommend changing the other ones either, to preserve easy inspection of line history with blame